### PR TITLE
Add some missing and incorrect items in networking code

### PR DESCRIPTION
**Pull request by euresti**
I noticed some inconsistencies between python2 and python3 in these modules.

### CODE REVIEW

File: stdlib/3/urllib/request.pyi [hunk 1]
```diff
@@ -140,9 +140,10 @@ class ProxyBasicAuthHandler(AbstractBasicAuthHandler, BaseHandler):
 
 class AbstractDigestAuthHandler:
     def __init__(self,
-                 password_mgr: Optional[HTTPPasswordMgr] = ...) -> None: ...
+                 passwd: Optional[HTTPPasswordMgr] = ...) -> None: ...
     def http_error_auth_reqed(self, auth_header: str, host: str, req: Request,
                               headers: Mapping[str, str]) -> None: ...
+    def get_authorization(self, req, chal): ...
```
Comment (ambv) [inline stdlib/3/urllib/request.pyi]: This is missing types.

**Comment by ambv** [comment]
General comment: In the future consider splitting PRs that touch different parts of the standard library, that will help us merge your changes faster. I don't have cycles today to confirm all the changes here. I'd merge 3/4 of this with no objections though.

File: stdlib/3/urllib/request.pyi (hunk 1 above)
Comment (euresti) [inline stdlib/3/urllib/request.pyi]: Yep. Same as in Python 2.  https://github.com/python/typeshed/blob/092b15e3e9ff376b7c141e137fcbd3397f08074a/stdlib/2/urllib2.pyi#L107

I can try to give it types but don't know how correct they'll be.

**Comment by euresti** [comment]
Splitting this.

**Pull request closed without merging**